docs/index.md

```diff
@@ -17,4 +17,11 @@ access PatternPal from `View > Other Windows > PatternPal Extension`.
 
 You can open the main view of the extension through `View > Other Windows > PatternPal Extension`.
 
+### Step-By-Step
+This module offers the flexibility for users to choose between implementing a design pattern from the beginning or continuing their work on a design pattern they have worked on previously. The supported design patterns can be selected from the dropdown menu. 
```

Comment:
This module offers the flexibility for users to choose between two options: starting the implementation of a new design pattern or continuing their work on a design pattern they have previously worked on.